repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/inspectors/ProjectInspector4Bears.java

```diff
@@ -55,13 +56,16 @@ public void run() {
             if (this.getBuildToBeInspected().getStatus() == ScannedBuildStatus.PASSING_AND_PASSING_WITH_TEST_CHANGES) {
                 cloneRepo.setNextStep(new CheckoutPatchedBuild(this, true, CheckoutPatchedBuild.class.getSimpleName()+"Candidate"))
                         .setNextStep(new ComputeSourceDir(this, true, true))
```

Comment:
Do you really need some steps "computeSourceDir" and "ComputeTestDir" to be blocking?

Reply:
For passing/passing, yes. If we are not able to compute the source dir, we can't do the checkout of the buggy build source code, and if we are not able to compute the test dir, we can't commit the tests separated.
But for failing/passing these things are not necessary (it's only to collect metrics as for REPAIR). I will change now for failing/passing.